<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogInDialog</class>
 <widget class="QDialog" name="LogInDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title_lb">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Super Chat</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_lb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_lb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_ledt">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入你的账号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwd_ledt">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入你的密码，至少6位</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="login_pbt">
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signup_pbt">
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exit_pbt">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="hint_Login_Info_lb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
